packages/strapi-admin/tests/admin-permission.test.e2e.js

```diff
@@ -115,7 +115,46 @@ describe('Role CRUD End to End', () => {
                   "label": "user",
                   "properties": Array [
                     Object {
-                      "children": Array [],
+                      "children": Array [
+                        Object {
+                          "label": "username",
+                          "required": true,
+                          "value": "username",
+                        },
+                        Object {
+                          "label": "email",
+                          "required": true,
+                          "value": "email",
+                        },
+                        Object {
+                          "label": "provider",
+                          "value": "provider",
+                        },
+                        Object {
+                          "label": "password",
+                          "value": "password",
+                        },
+                        Object {
+                          "label": "resetPasswordToken",
+                          "value": "resetPasswordToken",
+                        },
+                        Object {
+                          "label": "confirmationToken",
```

Comment:
Did we have all these in the previous version ?

Reply:
Yes, we use to have all those fields (password, tokens)

Reply:
eg: 3.5.1
![image](https://user-images.githubusercontent.com/25851739/116076161-2c16e280-a694-11eb-9134-aaf33aefa23e.png)